<launch>
  <!-- No namespace here as we will share this description. 
       Access with slash at the beginning -->
  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find m2wr_description)/urdf/m2wr.xacro'" />

  <!-- BEGIN ROBOT 1-->
  <group ns="Robot2">
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 1 -y 1 -z 0" />
      <arg name="robot_name"  value="Robot2" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2 -->
  <group ns="Robot6">
    <param name="tf_prefix" value="robot6_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -1 -y 1 -z 0" />
      <arg name="robot_name"  value="Robot6" />
    </include>
  </group>

  <group ns="Robot3">
    <param name="tf_prefix" value="robot3_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 1 -y -1 -z 0" />
      <arg name="robot_name"  value="Robot3" />
    </include>
  </group>

  <group ns="Robot7">
    <param name="tf_prefix" value="robot7_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -1 -y -1 -z 0" />
      <arg name="robot_name"  value="Robot7" />
    </include>
  </group>

  <group ns="Robot1">
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 2 -y 0 -z 0" />
      <arg name="robot_name"  value="Robot1" />
    </include>
  </group>

  <group ns="Robot4">
    <param name="tf_prefix" value="robot4_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 0 -y 2 -z 0" />
      <arg name="robot_name"  value="Robot4" />
    </include>
  </group>

  <group ns="Robot8">
    <param name="tf_prefix" value="robot8_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -2 -y 0 -z 0" />
      <arg name="robot_name"  value="Robot8" />
    </include>
  </group>

  <group ns="Robot5">
    <param name="tf_prefix" value="robot5_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 0 -y -2 -z 0" />
      <arg name="robot_name"  value="Robot5" />
    </include>
  </group>

  <group ns="Robot9">
    <param name="tf_prefix" value="robot9_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -5 -y 0 -z 0" />
      <arg name="robot_name"  value="Robot9" />
    </include>
  </group>

  <group ns="Robot10">
    <param name="tf_prefix" value="robot10_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 0 -y -5 -z 0" />
      <arg name="robot_name"  value="Robot10" />
    </include>
  </group>

  <group ns="Robot11">
    <param name="tf_prefix" value="robot11_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 0 -y 5 -z 0" />
      <arg name="robot_name"  value="Robot11" />
    </include>
  </group>

  <group ns="Robot12">
    <param name="tf_prefix" value="robot12_tf" />
    <include file="$(find orca)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x 5 -y 0 -z 0" />
      <arg name="robot_name"  value="Robot12" />
    </include>
  </group>
</launch>